### ISSUE TITLE
duality in xacro use_nominal_extrinsics

### ISSUE THREAD

**Issue by fherreroc** [realsenseai/realsense-ros#1064]
In `realsense2_description/urdf/_d435.urdf.xacro`, a use_nominal_extrinsics xacro:macro parameter is defined (`use_nominal_extrinsics:=false`) but it is not used, as later on line 75, a xacro argument is used instead (<xacro:if value="$(arg use_nominal_extrinsics)">).

If it is intended to be used as an argument, the xacro parameter is useless and can cause confusion, so it could be removed.

**Comment by RealSenseCustomerSupport** [comment]
----------------------------------------------

Hi fherreroc,

Actually it's used from the launch file here...

https://github.com/IntelRealSense/realsense-ros/blob/69d199d9e01a7c57b78831a6486c83ad8653aba9/realsense2_description/launch/view_d435_model.launch#L2

Here's the explanation from the original PR:

https://github.com/IntelRealSense/realsense-ros/pull/886

Thanks

**Comment by fherreroc** [comment]
Yes, in that launch file the **argument** is passed/used, but not the xacro **parameter** (which is defined but not used within _d435.urdf.xacro)
That's what I mean with "duality", it is defined in two ways, but only one is actually used. 
Also, the argument has no default value, so if you do not define it in your own launch file or xacro, you get the error "undefined substitution argument use_nominal_extrinsics None"

**Comment by mcamurri** [comment]
I noticed the same problem. It can be easily reproduced by running the test xacro:
```
xacro test_d435_camera.urdf.xacro
```
produces:
```
Undefined substitution argument use_nominal_extrinsics
when instantiating macro: sensor_d435 (/home/mcamurri/catkin_ws/src/realsense/realsense2_description/urdf/_d435.urdf.xacro)
in file: test_d435_camera.urdf.xacro
```

**Comment by mcamurri** [comment]
I've made a PR with the fix: #1125

**Comment by RealSenseSupport** [comment]
Hi @mcamurri,

Would you say we can close this now? 

FYI if we don’t hear back in 7 days, this issue will be closed.

Thanks again for your contribution!
